tests/Nelmio/Alice/Instances/Populator/PopulatorTest.php

```diff
@@ -22,7 +23,7 @@
 class PopulatorTest extends \PHPUnit_Framework_TestCase
 {
     const CONTACT = 'Nelmio\Alice\support\models\Contact';
-    const PLURAL = 'Nelmio\Alice\support\models\PluralProperties';
+    const PROPERTIES = 'Nelmio\Alice\support\models\Properties';
```

Comment:
No, sorry that was unclear - I would rather see constants for both the child class and the parent class and use them explicitly in the tests that require them so that it's clear what the intention is. For that matter, the `testProperty` method should probably be called `testSettingPrivatePropertiesDirectly`. Thoughts?